packages/mdc-select/component.ts

```diff
@@ -181,7 +181,9 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> {
   }
 
   set value(value: string) {
-    this.foundation_.setValue(value);
+    // Replicate behavior of native select, which does not emit change events
+    // when the value is changed programmatically.
+    this.foundation_.setValue(value, false /* notifyChange */);
```

Comment:
Since `setValue` is used for programmatic change of select's value and I think we do not need notifyChange boolean. 

Only `handleChange()` triggers event, where as `setValue()` does not.

WDYT?

Reply:
Good point. Removed the boolean from `setValue` so that it's only in `handleChange`.